CHANGELOG.md

```diff
@@ -1,3 +1,100 @@
+# [MongooseIM 6.1.0](https://github.com/esl/MongooseIM/releases/tag/6.1.0) - 2023-05-10
+
+## Highlights
+
+- Reworked C2S architecture
+- Docker image for arm64
+- Update of Universal Analytics to Google Analytics 4
+- Dialyzer types fix
+- Deprecation of Riak
+- More reliable test suites
+- Various improvements and fixes
+
+## All changes
+
+### Added
+- New C2S architecture (#3765)
+  - `presence` (#3746)
+  - `mod_last` (#3750)
+  - `mod_register` (#3751)
+  - `privacy` (#3747)
+  - `pubsub` (#3785)
+  - `ping` (#3748)
+  - `metrics` (#3800)
+  - `steam_management` (#3796)
+  - `mod_csi` (#3880)
+  - `sasl_external` (#3911)
+  - More granular hooks (#3852, #3857, #3955)
+  - Fixed `stream_management` timeouts (#3934)
+  - Stopped routing broadcast tuples (#3946)
+  - Unified metrics (#3967)
+  - C2S Migration guide (#3965)
+  - Fixed duplicated logout in `steam_management` (#3983)
+  - Adapted existing tests suites (#3772, #3778, #3783, #3787, #3786,  #3845, #3882, #3909, #3925, #3927, #3932, #3931, #3940)
+  - Miscellaneous changes (#3729, #3797, #3790, #3816, #3858, #3888, #3904, #3908, #3917, #3919, #3935, #3950, #3957, #3959, #3972)
+- Added information about GraphQL (#3905)
+- Enable CircleCI tests insights (#3899)
+- Hooks improvement (#3913, #3912)
+- `inbox` improvement (#3910)
```

Comment:
It might be good to group inbox extensions under one item - just grep for `inbox` in this list.